ui/src/app.tsx

```diff
@@ -43,12 +43,17 @@ const
       color: cssVar('$text'),
     },
     freeOverlay: {
-      display: 'none',
       position: 'fixed',
       left: 0, top: 0, right: 0, bottom: 0,
+      opacity: 0,
+      zIndex: -1,
     },
     busyOverlay: {
-      display: 'block',
+      position: 'fixed',
+      left: 0, top: 0, right: 0, bottom: 0,
+      opacity: 0.8,
+      zIndex: 1,
```

Comment:
Can we increase `zIndex` to some higher value (e.g. `999`) to make sure it won't get overriden by accident in future?

Reply:
Yes, sorry, I meant to but forgot.